CONFLICTS.md

```diff
@@ -70,3 +70,7 @@ references related issues.
 
     - https://github.com/stof/StofDoctrineExtensionsBundle/issues/455
     - https://github.com/doctrine-extensions/DoctrineExtensions/issues/2600
+
+- `doctrine/doctrine-bundle:2.11.0`:
+
+  This version uses the [readonly property](https://github.com/doctrine/DoctrineBundle/blob/2.11.0/Repository/ServiceEntityRepositoryProxy.php#L34), which is available from PHP 8.1 while the package also supports PHP 7.4 and 8.0, leading to compatibility issues.
```

Comment:
This really shouldn't be needed, the file you're linking is conditionally loaded by the ["real" class](https://github.com/doctrine/DoctrineBundle/blob/2.11.0/Repository/ServiceEntityRepository.php) downstream users are meant to be using, and this case it would be only when `doctrine/orm:^3.0` is installed (which requires PHP 8.1 anyway).  Sounds like this might be another case of a static analysis tool being over-eager?

Reply:
We managed to solve it in #15531. Thanks for rising this topic :)!